src/firefly/js/tables/ui/BasicTable.jsx

```diff
@@ -30,21 +28,18 @@ export class BasicTable extends React.Component {
             columnWidths: makeColWidth(props.columns, props.data, props.showUnits)
         };
 
-        this.onResize = this.onResize.bind(this);
+        this.onResize =  debounce((size) => {
+                if (size) {
+                    var widthPx = size.width;
+                    var heightPx = size.height;
+                    this.setState({widthPx, heightPx});
+                }
+            }, 100);
```

Comment:
The console logs gives more insight of what could happen:
TableStatsCntlr.js:178 Failed to fetch table statistics: TypeError: Cannot read property 'getNativeFromReactElement' of undefined
Please, have a look.